.rubocop.yml

```diff
@@ -44,13 +44,19 @@ PerlBackrefs:
     # we can't use named matches in 1.8.7
     - lib/generators/rspec/scaffold/scaffold_generator.rb
 
+HashSyntax:
+  Exclude:
+    # We probably can refactor the backref out, but for now excluding it since
+    # we can't use named matches in 1.8.7
+    - lib/generators/rspec/scaffold/scaffold_generator.rb
+
 Style/AccessModifierDeclarations:
   Enabled: false
 
 Naming/MemoizedInstanceVariableName:
   Enabled: false
 
-Naming/UncommunicativeMethodParamName:
```

Comment:
Which version of RuboCop do you have bundled? I guess 0.77.

Version [0.74 was supposed to be pinned](https://github.com/rspec/rspec-rails/blob/master/Gemfile#L70) 
still has [`UncommunicativeMethodParamName`](https://github.com/rubocop-hq/rubocop/blob/96b090f1a2b04fca2bf57b370f461ee5f1a4bd91/lib/rubocop/cop/naming/uncommunicative_method_param_name.rb#L46) ([also](https://github.com/rubocop-hq/rubocop/blob/96b090f1a2b04fca2bf57b370f461ee5f1a4bd91/config/default.yml#L2038)) and does not yet have `MethodParamName`.

Do you mind to change:
```diff
- gem 'rubocop', "~> 0.74"
+ gem 'rubocop', '0.74'
```
or
```diff
+ gem 'rubocop', "~> 0.74.0"
```
to avoid RuboCop errors due to an older version being bundled for other contributors?

Reply:
In this case, so many changes to `.rubocop*` files won't be required.

Reply:
Thanks @pirj , I will do update, thanks for reviewing.

Reply:
I'll handle this in #2216